builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -18,6 +18,7 @@ import groovy.transform.Field
 @Field def configLoader
 @Field def scmModule
 @Field def serviceConfigdata
+@Field def awsAPIGatewayModule
```

Comment:
Why we need apigateway module in build-pack lambda?

Reply:
this is needed to call a function getAPIhostnames() in gateway module, which in turn returns the ID for respective domain and stage

Reply:
Are we changing the jazz core service's deployment information

Reply:
Sorry I didn't get the meaning... We are just fetching the jazz_api_prod_id from this function and in turn we are using it for g_base_url 

g_base_url = https://${jazz_prod_api_id}.execute-api.${configLoader.AWS.DEFAULTS.REGION}.amazonaws.com/prod

Reply:
@saurav Why do we need API endpoint url when building lambda functions?

Reply:
this is being used for calling all the core services. Although the name of the function is confusing, I will make the respective change